test/e2e/e2e_suite_test.go

```diff
@@ -24,6 +25,7 @@ var (
 const (
 	testCatalogRefEnvVar = "CATALOG_IMG"
 	testCatalogName      = "test-catalog"
+	LatestImageTag       = "latest"
```

Comment:
Is there a reason to export this? I'd suggest unexporting:

```suggestion
	latestImageTag       = "latest"
```

Reply:
I'll hit this in a quick follow-up.

Reply:
I've made the changes in this PR itself